docs/fundamentals/executor/hub/push-executor.md

```diff
@@ -67,3 +67,15 @@ If you want to create a new tag for an existing Executor, you can also add the `
 ```bash
 jina hub push [--public/--private] --force-update <NAME> --secret <SECRET> -t TAG <path_to_executor_folder>
 ```
+
+### Immutable tags
+
+If you don’t want some tags to be later overwritten to keep a stable, consistent behavior, 
+immutable tags are the exact thing you are looking for.
+
+You can leverage the `--immutable-tag` option to create immutable tags. 
+After being pushed for the first time, the immutable tags can not be pushed again.
+
+```bash
+jina hub push [--public/--private] --force-update <NAME> --secret <SECRET> --immutable-tag <IMMUTABLE_TAG_1> --immutable-tag <IMMUTABLE_TAG_2> <path_to_executor_folder>
```

Comment:
My assumption for the usage of `immutable` was :point_down: . This would make sure, a certain tag is immutable. 
```bash
jina hub push [--public/--private] --force-update <NAME> --secret <SECRET> --tag v1 --immutable 
```

Not :point_down: . Any reason we want `tag` & `immutable-tag` both?

```bash
jina hub push [--public/--private] --force-update <NAME> --secret <SECRET> --immutable-tag <IMMUTABLE_TAG_1> --immutable-tag <IMMUTABLE_TAG_2> <path_to_executor_folder>
```

Reply:
Tag is an array, so there could be multiple `--tag`. 
I'm not familiar with `argparse`, Is there a proper way to distinguish which `--tag` the `--immutable` is for when there's multiple `--tag`?

Reply:
I'd say if `--immutable` is passed, we mark all tags immutable.

Reply:
@deepankarm There could be some situaction we want to use them both at the same time. For example a common scenerio, I want v1.2.3 immutable but v1 mutable. Then I can write 

`jina hub push --tag v1 --immutable-tag v1.2.3 .`